data/json/recipes/recipe_ammo.json

```diff
@@ -985,5 +985,17 @@
         [ "willowbark", 1 ]
       ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "cac2powder",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "60 m",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
```

Comment:
TODO but can wait, this needs to be finely ground and blended, like mortar and pestle operation, not just hammered.

Reply:
Once some feedback comes in about the system and some inevitable balancing happens, I'll modify the recipe and what's needed to accomplish it in a more logical fashion.   If I decide oxygen generation is feasibly something the survivor could accomplish in a reasonable manner, I might revisit some of the steps here.   For the time being, I'm going to wait on the chemistry overhaul project to progress before I look at more lab type processes.